how-to-release.md

```diff
@@ -84,65 +94,63 @@ for file in ./*.jar ; do shasum -a 1 $file > $file.sha1 ; done
 for file in ./*.jar ; do shasum -a 256 $file > $file.sha256 ; done
 ```
 
-*Note*: In the future, we might want to automate the generation of checksums via gradle.
+>**Note**: Generation of checksums via gradle is planned to be automated in the future releases.
 
-## Step 3: Upload the files to the pre-release draft
+### Step 3: Upload the files to the pre-release draft
 
 In the pre-release draft on GitHub, upload all the jar files (and its checksums) under `~/.m2/repository/io/pravega/pravega-connectors-flink_2.11/<RELEASE-VERSION>`. Follow the instructions on the page, it is straightforward.
 
-## Step 4: Release notes
+### Step 4: Release Notes
 
 Create a release notes text file containing the following:
-1. Some introductory text, highlighting the important changes going in the release.
-2. A full list of commits that you can get with the following command:
+1. Related introductory text, highlighting the important changes featured in the release.
+2. A complete list of commits can be obtained using the following command:
 ```
    > git log <commit-id-of-last-release>..<current-commit-id>
 ```
 
-* `<commit-id-of-last-release>` depends on the kind of release you are doing. If it is bug-fix release, then we can use the tag of the last branch release. For new branches, we have been adding `branch-X.Y` tags at the branch point for convenience. 
-* `<current-commit-id>` is the commit point of the release candidate you are working on. If you have manually tagged the release candidate, then you can go ahead and use it in the log command above.

```

Comment:
in the sentence: 
` the log command above.`
the term Above is not clear?where??